stdlib/REPL/src/emoji_symbols.jl

```diff
@@ -650,41 +933,70 @@ const emoji_symbols = Dict(
     "\\:runner:" => "🏃",
     "\\:running_shirt_with_sash:" => "🎽",
     "\\:sa:" => "🈂",
+    "\\:safety_pin:" => "🧷",
+    "\\:safety_vest:" => "🦺",
     "\\:sagittarius:" => "♐",
     "\\:sake:" => "🍶",
+    "\\:salt:" => "🧂",
     "\\:sandal:" => "👡",
+    "\\:sandwich:" => "🥪",
     "\\:santa:" => "🎅",
+    "\\:sari:" => "🥻",
     "\\:satellite:" => "📡",
+    "\\:satellite_antenna:" => "📡",
+    "\\:sauropod:" => "🦕",
     "\\:saxophone:" => "🎷",
+    "\\:scarf:" => "🧣",
     "\\:school:" => "🏫",
     "\\:school_satchel:" => "🎒",
     "\\:scissors:" => "✂",
+    "\\:scooter:" => "🛴",
+    "\\:scorpion:" => "🦂",
     "\\:scorpius:" => "♏",
     "\\:scream:" => "😱",
     "\\:scream_cat:" => "🙀",
+    "\\:screwdriver:" => "🪛",
     "\\:scroll:" => "📜",
+    "\\:seal:" => "🦭",
     "\\:seat:" => "💺",
+    "\\:second_place_medal:" => "🥈",
     "\\:secret:" => "㊙",
     "\\:see_no_evil:" => "🙈",
     "\\:seedling:" => "🌱",
+    "\\:selfie:" => "🤳",
+    "\\:sewing_needle:" => "🪡",
+    "\\:shallow_pan_of_food:" => "🥘",
+    "\\:shark:" => "🦈",
     "\\:shaved_ice:" => "🍧",
     "\\:sheep:" => "🐑",
     "\\:shell:" => "🐚",
     "\\:ship:" => "🚢",
     "\\:shirt:" => "👕",
+    "\\:shopping_trolley:" => "🛒",
+    "\\:shorts:" => "🩳",
     "\\:shower:" => "🚿",
+    "\\:shrimp:" => "🦐",
+    "\\:shrug:" => "🤷",
+    "\\:shushing_face:" => "🤫",
     "\\:signal_strength:" => "📶",
     "\\:six_pointed_star:" => "🔯",
+    "\\:skateboard:" => "🛹",
     "\\:ski:" => "🎿",
-    "\\:skin-tone-2:" => "\U1f3fb",
-    "\\:skin-tone-3:" => "\U1f3fc",
-    "\\:skin-tone-4:" => "\U1f3fd",
-    "\\:skin-tone-5:" => "\U1f3fe",
-    "\\:skin-tone-6:" => "\U1f3ff",
```

Comment:
These are the same values.
I am not sure what changed that changed how they are made into a string.